blog/_posts/2019-04-24-MLJ.md

```diff
@@ -0,0 +1,253 @@
+<!---
+![](https://github.com/JuliaLang/www.julialang.org/tree/master/images/blog/2019-04-26-MLJ/learningcurves.png)
+![](https://github.com/JuliaLang/www.julialang.org/tree/master/images/blog/2019-04-26-MLJ/heatmap.png)
+![](https://github.com/JuliaLang/www.julialang.org/tree/master/images/blog/2019-04-26-MLJ/packages.jpg)
+![](https://github.com/JuliaLang/www.julialang.org/blob/master/images/blog/2019-04-26-MLJ/)
+Images to be placed
+-->
+<!-- Diagrams drew with with http://dia-installer.de -->
+# Beyond machine learning pipelines with MLJ
+
+Anthony Blaom, Diego Arenas, Franz Kiraly, Yiannis Simillides, Sebastian Vollmer
+
+<!---
+- learning curves (from `examples/random_forest.jl`)
+- tuning plots (from `examples/two_parameter_plot.jl`)
+-->
+<p float="left" align="middle">
+  <img src="https://github.com/JuliaLang/www.julialang.org/blob/master/images/blog/2019-04-26-MLJ/learningcurves.png" width="400" />
+  <img src="https://github.com/JuliaLang/www.julialang.org/blob/master/images/blog/2019-04-26-MLJ/heatmap.png" width="400" /> 
+</p>
+
+<p float="left" align="middle">
+  <img src="https://github.com/JuliaLang/www.julialang.org/blob/master/images/blog/2019-04-26-MLJ/wrapped_ridge.png" width="400" />
+  <img src="https://github.com/JuliaLang/www.julialang.org/blob/master/images/blog/2019-04-26-MLJ/MLPackages.png" width="400" /> 
+</p>
+
+
+## Introduction
+
+[MLJ](https://github.com/alan-turing-institute/MLJ.jl) is an
+open-source machine learning toolbox written in pure Julia. It
+provides a uniform interface for interacting with supervised and
+unsupervised learning models currently scattered in different Julia
+packages.
+
+Building on a earlier proof-of-concept, development began in earnest
+at [The Alan Turing Institute](https://www.turing.ac.uk) in
+December 2018. In a short time interest grew and the project is now
+the Institute's most starred software repository.
+
+After outlining MLJ's current functionality, this post introduces MLJ
+**learning networks**, a super-charged pipelining feature for model
+composition.
+
+**Quick links:**
+
+&#9758; [MLJ vs ScikitLearn.jl](https://alan-turing-institute.github.io/MLJ.jl/dev/frequently_asked_questions/)  
```

Comment:
Should these just be - ?

Reply:
☞ is what Anthony wanted, but they can be changed if it would make more sense / better rendering on all browsers